commons/weather/weather_current.avsc

```diff
@@ -0,0 +1,32 @@
+{
+  "namespace": "org.radarcns.weather",
+  "type": "record",
+  "name": "WeatherCurrent",
+  "doc": "Weather information at the time reported.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "sunrise", "type": "double", "doc": "sunrise time in hours after midnight (h). The difference with sunset is the hours of daylight."},
+    {"name": "sunset", "type": "double", "doc": "sunset time in hours after midnight (h). The difference with sunrise is the hours of daylight."},
+    {"name": "temperature", "type": ["null","float"], "doc": "current ambient temperature (C)"},
```

Comment:
please add `"default": null` for for these and previous and following fields (except the `enum` and `source` fields).